jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -29,6 +29,7 @@ val ENUM_TABLES =
                 EnumTable("document_stores", isLocalizable = false),
                 EnumTable("event_statuses"),
                 EnumTable("event_types"),
+                EnumTable("internal_interests"),
```

Comment:
Probably safe to make this `isLocalizable = false` since it's only for internal use. Then you don't need it in `Enums.properties`.